### ISSUE TITLE
create_report Fails for a model that did not converge

### ISSUE THREAD

**Issue by billdenney** [pharmpy/pharmpy#66]
I'm trying out `pharmpy` via R, and I was playing around with what is available.

I tried to run the `create_report()` method for the `modelfit_results` of a `Model()`.  The model had termination errors.  While doing it, I got an error:

```
> models[[1]]$modelfit_results$create_report("test")
 Error in py_call_impl(callable, dots$args, dots$keywords) : 
  AttributeError: 'NONMEMChainedModelfitResults' object has no attribute 'rst_path' 
```

**Comment by rikardn** [comment]
Thanks for reporting this. There is no report available for a regular modelfit yet. We need to generate a better error message.

**Comment by billdenney** [comment]
Yeah, this was just me trying out what seemed to be available.  It makes sense that it's not here yet.
